examples/internal/proto/examplepb/BUILD.bazel

```diff
@@ -8,14 +8,24 @@ package(default_visibility = ["//visibility:public"])
 # TODO(yannic): Add examples/tests that use import_prefix/strip_import_prefix.
 
 # gazelle:exclude a_bit_of_everything.pb.gw.go
+# gazelle:exclude a_bit_of_everything_grpc.pb.go
```

Comment:
This is still a bit of a hack, see https://github.com/bazelbuild/bazel-gazelle/issues/798